modules/serverless-logic-persistence-precedence-rules.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-persistence
+
+
+:_mod-docs-content-type: REFERENCE
```

Comment:
In my opinion this is more of a conceptual module because it explains concepts in depth, whereas a reference module is more like something with information that users can quickly reference if needed, like parameters and such. See the mod docs guidelines definitions for [concept modules](https://redhat-documentation.github.io/modular-docs/#concept-module-definition) and [reference modules](https://redhat-documentation.github.io/modular-docs/#reference-module-definition).
```suggestion
:_mod-docs-content-type: CONCEPT
```

Reply:
Done